mycqu/card.py

```diff
@@ -201,5 +289,37 @@ def get_fee_data(synjones_auth, room, fee_item_id):
     if dic['msg'] == 'success':
         return dic
     else:
-        raise FeeAcquisitionFailed(dic['msg'])
+        raise AcquisitionFailed(dic['msg'])
+
+
+# 获取校园卡账号信息
+def get_card_data(session):
+    url = "http://card.cqu.edu.cn/NcAccType/GetCurrentAccountList"
+
+    res = session.post(url)
+    result = json.loads(json.loads(res.text))
+    if result['respCode'] != "0000":
+        raise AcquisitionFailed(error_msg=result['respInfo'])
+
+    return result['objs']
+
+
+# 获取消费账单记录
+def get_bill_data(session, account, duration):
```

Comment:
同上这个似乎更像 `get_bill_raw`
```suggestion
def get_bill_raw(session, account, duration):
```